uv.gyp

```diff
@@ -484,7 +484,10 @@
           'sources': [
             'test/runner-unix.c',
             'test/runner-unix.h',
-          ]
+          ],
+          'link_settings': {
+            'libraries': [ '-lutil' ],
+          },
```

Comment:
This is for the run-benchmarks target, you need to move it above, to the run-tests target.